common/tensorrt_common/CMakeLists.txt

```diff
@@ -16,6 +16,45 @@ if(NOT (CUDAToolkit_FOUND AND CUDNN_FOUND AND TENSORRT_FOUND))
   return()
 endif()
 
+# Download onnx
```

Comment:
We decided not to include download scripts in CMake. 
Maybe we could skip TestGetInputDims for now.

Reply:
To be honest, this package is not really used in autoware.

Reply:
Are you sure about that? The header files seems to be included by the other packages. ([example](https://github.com/autowarefoundation/autoware.universe/blob/0dee51263a607ea19f99f9de34e26668368547a1/perception/tensorrt_yolox/include/tensorrt_yolox/tensorrt_yolox.hpp#L21))

I will discuss this in the next Software WG if we could consider adding sample models in the repository or not for tests.

Reply:
> To be honest, this package is not really used in autoware.

@mitsudome-r 
sorry, I didn't make it clear, there are 3 packages in the  perception folder related to tensorrt: `tensorrt_classifier` , `tensorrt_yolo`, `tensorrt_yolox`, where both `tensorrt_classifier`  and `tensorrt_yolox` contain the `tensorrt_common`, but `tensorrt_yolo` does not contain it.  for our real car tests we only used the `tensorrt_yolo`.